.eslintrc

```diff
@@ -1,15 +1,190 @@
 {
-  "parser": "babel-eslint",
-  "extends": "airbnb/base",
```

Comment:
yep. The .eslintrc is largely based on airbnb. I removed it due to some conflicting rules. If someone can resolve the conflicts, there's no reason not to add it in. But what's there is pretty comprehensive.